RestSharpClient/ApiClient/ApiMethodParams.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace RestSharpClient.ApiClient
+{
+	public class ApiMethodParams
+	{
```

Comment:
please format your code before requesting a review

Reply:
fixed the misformatted codes across the project and using 4 spaces default.